public/recruiting.js

```diff
@@ -22,10 +25,20 @@ var requestTimeout = (evt) => {
     console.log(evt.target);
 };
 
+var clearBalls = () => {
+    var cursors = document.getElementsByClassName("cursor");
```

Comment:
Minor nitpick: I'd go with `document.querySelectorAll(".cursor")` for consistency regarding other DOM queries in this script.